### PR TITLE
Announcing Scala.js 0.6.29.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

File: _posts/news/2019-09-18-announcing-scalajs-0.6.29.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## Bug fixes
+
+Among others, the following bugs have been fixed in 0.6.29:
+
+* [#3419](https://github.com/scala-js/scala-js/issues/3419) / [#3596](https://github.com/scala-js/scala-js/issues/3596) Major bug in incremental compilation with sbt 1.x
+* [#3607](https://github.com/scala-js/scala-js/issues/3607) Calling `getClass.getSimpleName` on objects returns name without trailing `$`
+* [#3604](https://github.com/scala-js/scala-js/issues/3604) Implement `java.util.Collection.removeIf`
+* [#3605](https://github.com/scala-js/scala-js/issues/3605) `ArrayBuilder.generic.addAll` is broken on 2.13
+
+You can find the full list [on GitHub](https://github.com/scala-js/scala-js/issues?q=is%3Aissue+milestone%3Av0.6.29+is%3Aclosed).
```
Comment (gzm0) [inline _posts/news/2019-09-18-announcing-scalajs-0.6.29.md]: The following commits are entirely untracked but are bugfixes. This is not good.

https://github.com/scala-js/scala-js/commit/f0a0228ebf0f50234411157777e30fba22ce4741
https://github.com/scala-js/scala-js/commit/ed5d5f4334303b7db1e12eabc9f590e8506feae7
https://github.com/scala-js/scala-js/commit/5180897c2f9d8c5200a5b41bfd9892a20d6b755e
https://github.com/scala-js/scala-js/commit/cde963b2638120f07304ccad6e870dd6bb9f28ea
https://github.com/scala-js/scala-js/commit/52ac12ca00576cb6574b61de923d71de29c697c6

File: _posts/news/2019-09-18-announcing-scalajs-0.6.29.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+val moduleName = "foo.js"
+val promise = js.`import`[FooAPI](moduleName)
+val future = promise.toFuture
+for (module <- future) {
+  println(module.bar(5))
+}
+{% endhighlight %}
+
```
Comment (gzm0) [inline _posts/news/2019-09-18-announcing-scalajs-0.6.29.md]: Other stuff we should mention somewhere?
- test-interface / test-bridge split (also put it in the artifacts?)
- unicode update
- stack trace suppression of LinkingException
- errors on duplicat exports
- `globalThis` deprecation

Comment (sjrd) [reply _posts/news/2019-09-18-announcing-scalajs-0.6.29.md]: I don't think the stack trace suppression is worth mentioning in the release notes.

I have added everything else you mention.

File: _posts/news/2019-09-18-announcing-scalajs-0.6.29.md (hunk 1 above)
Comment (sjrd) [reply _posts/news/2019-09-18-announcing-scalajs-0.6.29.md]: I have created issues and closed them *a posteriori*, so that they can be tracked.

**Comment by sjrd** [comment]
Ping @gzm0?

**Pull request merged**